.github/templates/macos_binary_build_workflow.yml.j2

```diff
@@ -22,6 +22,14 @@ name: !{{ build_environment }}
           echo "MAC_PACKAGE_WORK_DIR=${RUNNER_TEMP}" >> "${GITHUB_ENV}"
 {%- endmacro %}
 
+{%- macro setup_python(py_ver) -%}
+      - name: Setup Python
+        uses: actions/setup-python@v5
```

Comment:
FYI, setup-python@v6 is out: https://github.com/actions/setup-python/releases/tag/v6.0.0